rosdep/base.yaml

```diff
@@ -6578,6 +6578,15 @@ usbutils:
     homebrew:
       packages: [mikhailai/misc/usbutils]
   ubuntu: [usbutils]
+util-linux:
+  arch: [util-linux]
+  debian:
+    buster: [util-linux]
```

Comment:
Why is this only listed for buster? 

It appears to be available for all active distros: 
https://packages.debian.org/buster/util-linux

Reply:
I totally missed those @tfoote, thanks!